docs/tasks.md

```diff
@@ -1,5 +1,21 @@
 # What can it do?
 
+List of all the keys under `config.tasks`:
```

Comment:
Please include this table under `Autopilot.` as it already contains a sample config.

Reply:
Hope I did it right.

Actually I prefer it to be at top, because it is a kind of menu.